alert/search.go

```diff
@@ -41,6 +45,22 @@ type SearchCursor struct {
 }
 
 var searchTemplate = template.Must(template.New("search").Parse(`
+	{{ if .IncludeNotifiedUser }}
+		SELECT
+			a.id,
+				a.summary,
+				a.details,
+				a.service_id,
+				a.source,
+				a.status,
+				created_at,
+				a.dedup_key
+		FROM alerts a
+		JOIN alert_logs al ON al.alert_id = a.id AND al.event = 'notification_sent'
+		WHERE al.sub_user_id = :currentUserID
```

Comment:
This should be mixed into the where clause -- It needs to respect Omit, Search, Status, pagination etc.. The only filter it should override is the service id restriction

```
	{{ if .Services }}
		AND (a.service_id = any(:services) OR sub_user_id)
	{{ end }}
```

Reply:
I originally had this in the where clause but was running into some issues with aggregating the notified alerts on top of the logic currently in place. I'll keep brainstorming

Reply:
Just updated without using the ternary, let me know your thoughts once you get a chance @mastercactapus!